gdquest_art_tools/GDquestArtTools.py

```diff
@@ -64,6 +65,30 @@ def exportSelectedLayers(cfg, statusBar):
         msg, timeout = cfg['error']['msg'].format(e), cfg['error']['timeout']
     statusBar.showMessage(msg, timeout)
 
+def exportCOATools(cfg, statusBar):
+    msg, timeout = cfg['done']['msg'].format('Exported selected layers.'), cfg['done']['timeout']
+    try:
+        doc = KI.activeDocument()
+        ensureRGBAU8(doc)
+
+        coat_format = COAToolsFormat(cfg, statusBar)
+        dirName = osp.dirname(doc.fileName())
+        nodes = KI.activeWindow().activeView().selectedNodes()
+
+        # By convention all selected nodes should be Group Layers
+        for n in nodes:
+            wn = WNode(cfg, n)
+            if not wn.isGroupLayer():
+                raise ValueError(wn.name,'is not a Group Layer')
+
+        it = map(partial(WNode, cfg), nodes)
+        it = map(lambda n: coat_format.collect(n), it)
```

Comment:
this can be simplified to `map(coat_format.collect, it)` since it's a one-parameter function

Reply:
Good catch :) - fixed in 86c78f9